src/AutoRest.CSharp/Common/Output/Builders/SerializationBuilder.cs

```diff
@@ -193,6 +196,44 @@ public static JsonSerialization BuildSerialization(Schema schema, CSharpType typ
             }
         }
 
+        public static BicepSerialization BuildBicepSerialization(Schema schema, CSharpType type, bool isCollectionElement)
+        {
+            switch (schema)
```

Comment:
this could aslo be a switch expression I think?

Reply:
This method will go away